### DISCUSSION TITLE
Mark Searches as Private in Search Results

### DISCUSSION THREAD

**Discussion by dd-manderson** [burkybang/SuiteAdvanced-Feedback#138] [Ideas]
Would be handy if you were able to indicate if a search was private or not in the search results when searching for saved searches.  

![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/160878598/22722044-0b99-48ba-8a8f-b2e4f1e53adf)

**Comment by burkybang** [comment]
I can think of a way I'd accomplish that, but I'd be concerned about performance. It would have to look up each search to see if it's private or not. I'm not sure if the benefit of the information would outweigh the performance impact.

**Reply by dd-manderson** [reply]
I was thinking the same thing.  I wasn't sure if the underlying JSON in the search results returned a private flag.  Doesn't sound like it.  I agree the performance hit of looking this up in real-time would be too much.
